draft-ietf-scitt-architecture.md

```diff
@@ -376,6 +376,23 @@ The SCITT architecture consists of a very loose federation of Transparency Servi
 In order to accommodate as many Transparency Service implementations as possible, this document only specifies the format of Signed Statements (which must be used by all Issuers) and a very thin wrapper format for Receipts, which specifies the Transparency Service identity and the agility parameters for the Signed Inclusion Proofs.
 Most of the details of the Receipt's contents are specified in the COSE Signed Merkle Tree Proof document {{-COMETRE}}.
 
+SCITT consists of the following roles: Issuers, Transparency Services, Verifiers, and Auditors.
+The process works as follows:
+
+1. Issuers make Statements about Artifacts
+1. Issuers sign the Statements, producing Signed Statements
+1. Issuers submit the Signed Statements to Transparency Services
+1. Transparency Services take the submitted Signed Statements and insert them into an Append-only Log, producing Transparent Statements.
+This process can be thought of as Transparency Services counter-signing Issuers' Signed Statements
+1. Verifiers consume the Transparent Statements, and by verifying them, are convinced of the authenticity and integrity of Statements about an Artifact
```

Comment:
```suggestion
1. Verifiers query a Transparency Service for one or more Transparent Statements associated with a Subject
1. By verifying the Transparent Statement payloads, the verifier may be convinced to the authenticity and integrity of an Artifact
```

Reply:
@SteveLasker Your suggestion at line 387 is not correct. A Verifier need not and should not query the Transparency Service. That is the whole point of the Append only log is that N- Independent Entities (receiving Transparent Statements) need to need to check the log, but just use the Transparent statment possessing the receipt to Verify the presence of the Statement in the log.

Reply:
@yogeshbdeshpande, I believe we're mixing multiple scenarios here. 
An issuer registers Signed Statements and gets a Receipt. How does a verifier get all the receipts?
A verifier may not have access to the Receipts, and simply wants to get the current status for an artifact. What Statements have been made by Issuers they trust? Have any of the Issuers made amended statements that should be known? By querying the feed, based on the subject, they can see a wholistic picture and decide if the artifact meets their needs.

Reply:
@thgoebel accept or dismiss this suggestion please : )

Reply:
> 1. By verifying the Transparent Statement payloads, the verifier may be convinced to the authenticity and integrity of an Artifact

This means effectively verifying `Statement playload`, which is not a thing. Did you mean `Sign Statement` instead of `Transparent Statement payloads`?

Reply:
I think @SteveLasker's main concern is having a bullet on how the Verifier obtains the Transparent Statement?
Maybe we can list the different options here. Querying the TS, getting it from the Issuer directly, any others?

Reply:
> I think @SteveLasker's main concern is having a bullet on how the Verifier obtains the Transparent Statement?
Maybe we can list the different options here. Querying the TS, getting it from the Issuer directly, any others?

Okay. But how is that related to the new item?

> 1. By verifying the Transparent Statement payloads, the verifier may be convinced to the authenticity and integrity of an Artifact

Reply:
This "new" item is just my proposal pulled down and slightly edited ("payload" added, "s/of Statements about an Artifact/of an Artifact"). They are related in that you first need to obtain the Transparent Statement before you can verify it :)

I pushed an update for this.

Reply:
oh! I misread the suggestion scope. Sorry. So my question was targeted at you in the end! I'll chime in again, if this item stalls.